Assets/MixedRealityToolkit.Examples/Experimental/SpatialAwareness/.Documentation/BytesInDevicePortal.png.meta

```diff
@@ -0,0 +1,88 @@
+fileFormatVersion: 2
+guid: f5d74b997f52d224ea01b4b80ffd492e
```

Comment:
We've typically stuck docs images in the Images folder in the root Documentation folder. Would it be possible to move these there?

Reply:
Moved the image location & updated .md file's path information.